sonar/modules/collections/config.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#
+# Swiss Open Access Repository
+# Copyright (C) 2021 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Configuration."""
+
+from sonar.modules.permissions import record_permission_factory
+
+# Resource name
+RESOURCE_NAME = 'collections'
+
+# JSON schema name
+JSON_SCHEMA_NAME = RESOURCE_NAME[:-1]
+
+# Module path
+MODULE_PATH = f'sonar.modules.{RESOURCE_NAME}'
+
+# PID type
+PID_TYPE = 'coll'
+
+
+class Configuration:
+    """Resource configuration."""
+
+    index = f'{RESOURCE_NAME}'
+    schema = f'{RESOURCE_NAME}/{JSON_SCHEMA_NAME}-v1.0.0.json'
+    pid_type = PID_TYPE
+    resolver_url = f'/api/{RESOURCE_NAME}/<pid>'
+    rest_endpoint = {
```

Comment:
I don't like PEP-8 80 characters line limit. Not easy to read :-)

Reply:
Completely agree with you, we should increase this limit.